src/test/java/org/fundacionjala/coding/wilson/CCTest.java

```diff
@@ -0,0 +1,19 @@
+package org.fundacionjala.coding.wilson;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class CCTest {
+
+    /**
+     *
+     */
+    @Test
+    public void testSomething() {
+        final int expectedResult = 4;
+        assertEquals(expectedResult, CC.charCount("fizzbuzz", 'z'));
```

Comment:
Your implementation is failing for this unit test
assertEquals(expectedResult, CC.charCount("fizzbuZZ", 'z'));